modular_robot_physical/revolve2/modular_robot_physical/physical_interfaces/v1/_v1_physical_interface.py

```diff
@@ -11,36 +13,53 @@ class V1PhysicalInterface(PhysicalInterface):
     _PWM_FREQUENCY = 50
     _CENTER = 157.0
     _ANGLE60 = 64.0
+    _PINS = list(range(2, 28))
```

Comment:
are the pins always in this range?

Reply:
that is what the internet says. i will test on real hardware next tuesday

Reply:
it appears to be fine.